docker-compose.yaml

```diff
@@ -120,6 +120,51 @@ services:
         ipv4_address: 172.16.238.30
     restart: always
 
+  devnet-server-1:
+    image: hydra-auction-delegate:latest
```

Comment:
So, I am missing:

* Where does building take place?
* Where do it stores images? Locally I presume?

Does our CI automatically call whose derivations building images?

Reply:
Could you please explain how to call building locally.